### PR TITLE
Allow rendering pre-sorted tables 

### PR DESCRIPTION
**Pull request by karthikbtf**

### CODE REVIEW

**Comment by karthikbtf** [comment]
using a minor label here

File: src/components/table/index.tsx [hunk 1]
```diff
@@ -176,6 +188,13 @@ export default function EnhancedTable<T extends TableRowType>({
     if (sortHandler) {
       sortHandler(newOrder, property);
     }
+
+    // We want to set the page to 1 on sort order change for presorted tables but we won't have
```
Comment (constanzauanini) [inline src/components/table/index.tsx:192]: remove this maybe?

File: src/components/table/index.tsx [hunk 2]
```diff
@@ -167,6 +169,16 @@ export default function EnhancedTable<T extends TableRowType>({
     }
   }, [rows, setSelectedRows]);
 
+  useEffect(() => {
+    // this is not most elegant but we want to do this if table was sorted by some column in a presorted table
+    // but we don't know when the data changes, hence this useEffect on the data size
+    if (rows.length && isPresorted) {
+      // we want to set page back to 1 if the data changes on presorted lists,
+      // this is because the data was reset due to some sort behavior refetching new data
+      handleChangePage({}, 1);
```
Comment (constanzauanini) [inline src/components/table/index.tsx:178]: Don't we need to execute this for any row change? (Not only a length change)

Comment (karthikbtf) [reply src/components/table/index.tsx:178]: ugh good point, the number of rows may stay the same...

Comment (karthikbtf) [reply src/components/table/index.tsx:178]: updated, thanks for catching this..

**Review by constanzauanini — APPROVED** [review]
Looks good!

**Pull request merged**